runtime/onert/core/include/ir/operation/ResizeBilinear.h

```diff
@@ -34,10 +34,12 @@ class ResizeBilinear : public Operation
   enum Input
   {
     INPUT = 0,
+    SIZE = 1,
   };
 
   struct Param
   {
+    // If the input SIZE exists in inputs, height_out and width_out are not set. Ignore these params
```

Comment:
`height_out` and `width_out` are used in nnapi frontend only, am I right?
Is it possible to create a new tensor (size) from these two variables in the nnapi frontend and then remove them?

Reply:
It is not simple. According to nnapi spec, `height_out` and `width_out` each have their own index. This makes it is not simple to create a new tensor from these two operands.
First, if `height_out` and `width_out` are constant, their values will be set by the api `ANeuralNetworksModel_setOperandValue` respectively. But there is no operation information in the api. This means that runtime should find and match them to a new operand (size). If IR of this operation doesn't have `height_out` and `width_out`, where should runtime works it within frontend.
Second, if `height_out` and `width_out` are non-constant, it's really complicated.Their values will be set by the api `ANeuralNetworksExecution_setInput` or outputs of other operations respectively. `ANeuralNetworksExecution_setInput` don't have operation information too. This would cause a similar problem to the first. Furthermore, the works to find and match them make the performance degradation in execution time. The critical problem is the case when these are when they are output of the other operations. I can't even think of them as the output of other operations because this operation has them as a input(size) in model of tflite, But nnapi treats it as two inputs. I think this means that the two operands never be non-constant or that nnapi has a spec error.